### ISSUE TITLE
Close connection from server-side

### ISSUE THREAD

**Issue by parnham** [asleepysamurai/socketwrench#8]
Platform: Linux (Ubuntu 18.04)
Version: 0.0.1

When the websocket server attempts to close the connection either through a message or when the server-side application exits the connection doesn't appear closed in Socket Wrench. If I then try to disconnect within the Socket Wrench interface I get the following error:

```
An unexpected error occurred while trying to disconnect. Please try again later, and if the issue persists, contact Socket Wrench Support.
```

I have to either close the application or start a new connection (leaving the previous one there but non-functional).

**Comment by asleepysamurai** [comment]
Yup, seems like a bug, and I can reproduce it. Will add to the list of stuff to fix. Unfortunately, I'm a bit busy these days, so it might be a while before a fix is available.
